stats/stats.go

```diff
@@ -135,11 +135,13 @@ var (
 		2000, 3000, 4000, 5000, 7500, 10000, // another order of magnitude coarsly covered
 		20000, 30000, 40000, 50000, 75000, 100000, // ditto, the end
 	}
-	numValues  = len(histogramBucketValues)
-	numBuckets = numValues + 1 // 1 special first bucket is <= 0; and 1 extra last bucket is > 100000
-	firstValue = float64(histogramBucketValues[0])
-	lastValue  = float64(histogramBucketValues[numValues-1])
-	val2Bucket []int
+	numValues          = len(histogramBucketValues)
+	numBuckets         = numValues + 1 // 1 special first bucket is <= 0; and 1 extra last bucket is > 100000
+	firstValue         = float64(histogramBucketValues[0])
+	lastValue          = float64(histogramBucketValues[numValues-1])
+	maxArrayValue      = int32(1000) // Last value looked up as O(1) array, the rest is linear search
```

Comment:
ping on inserting a new line before this line
and leaving val2Bucket  in the first set

Reply:
Okay I do this now. But I couldn't figure out to do white space stuff with re-go-fmt - tia.

Reply:
if you insert a line, go fmt won't try to make it all aligned and thus it will reduce the diff

if your editor doesn't do it out of the box (I recommend you get a better editor: Atom + go-plus is free and great) you can run goimports or gofmt from the command line

Reply:
make lint ask me to format them. It still causes to be diff.

Reply:
btw, I'm using GoLand